package/pagesource-code-quality/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "pagesource-code-quality",
```

Comment:
There needs to be separate packages for each type of quality check. 
They will have a dedicated repo for it.  
In this repo it will only come as dependency and a config file to apply them